packages/origin-backend/src/pods/device/device.controller.ts

```diff
@@ -46,11 +46,26 @@ export class DeviceController {
         return this.deviceService.getAll();
     }
 
+    @Get('/myDevice')
+    @UseGuards(AuthGuard(), RolesGuard)
+    @Roles(Role.OrganizationAdmin, Role.OrganizationDeviceManager, Role.OrganizationUser)
+    async getMyDevice(@UserDecorator() { organizationId }: ILoggedInUser) {
```

Comment:
The method is called `getMyDevice` but it fetches device**s**. Please change this to `getMyDevices()`.